packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -163,15 +248,56 @@ export default class EccUtilsDesignForm extends LitElement {
         </div>
       `;
     }
-
+  
+    if (field.type === "select") {
+      return html`
+        <div class="select-container">
+          ${field.fieldOptions?.tooltip && field.fieldOptions.tooltip !== ""
+            ? html`
+                <sl-tooltip
+                  id=${field.key}
+                  content=${field.fieldOptions?.tooltip}
+                >
+                  <label class="select-label">
+                    ${field.label} ${field.fieldOptions?.required ? "*" : ""}
+                  </label>
+                </sl-tooltip>
+              `
+            : html`
+                <label class="select-label">
+                  ${field.label} ${field.fieldOptions?.required ? "*" : ""}
+                </label>
+              `}
+          <sl-select
+            class="select"
+            ?required=${field.fieldOptions?.required}
+            value=${_.get(this.form, path)}
+            @sl-change=${(e: Event) => {
+              const { value } = e.target as HTMLSelectElement;
+              _.set(this.form, path, value);
+              this.requestUpdate();
+            }}
+          >
+            ${field.fieldOptions?.options?.map(
+              (option) => html`
+                <sl-option value=${option.value}>
+                  ${option.label}
+                </sl-option>
+              `
+            )}
+          </sl-select>
+        </div>
+      `;
+    }
+  
     if (!_.get(this.form, path)) {
```

Comment:
this if statement sets the default text or option for the input fields on the form, the code rendering the select field should be below it.

Reply:
The code rendering is below the if statement.

Reply:
it's not. Perhaps there is some confusion on which statement I am talking about. The statement I am talking about is this one: 
```
if (!_.get(this.form, path)) {
      if (field.fieldOptions?.default && !this.hasUpdated) {
        _.set(this.form, path, field.fieldOptions.default);
      } else if (field.fieldOptions?.returnIfEmpty) {
        _.set(this.form, path, '');
      }
    }
```

The code rendering the `select` field should be below that statement